pkg/query-service/model/v3/v3.go

```diff
@@ -256,10 +256,11 @@ const (
 )
 
 type AttributeKey struct {
-	Key      string               `json:"key"`
-	DataType AttributeKeyDataType `json:"dataType"`
-	Type     AttributeKeyType     `json:"type"`
-	IsColumn bool                 `json:"isColumn"`
+	Key               string               `json:"key"`
+	DataType          AttributeKeyDataType `json:"dataType"`
+	Type              AttributeKeyType     `json:"type"`
+	IsColumn          bool                 `json:"isColumn"`
+	IsDefaultEnriched bool                 `json:"isDefaultEnriched"`
```

Comment:
No need to use `json:"isDefaultEnriched"` as we are not parsing `IsDefaultEnriched` from a json.

Reply:
Have removed it.